core/arch/arm/kernel/thread.c

```diff
@@ -613,6 +614,31 @@ size_t thread_stack_size(void)
 	return STACK_THREAD_SIZE;
 }
 
+void get_stack_limits(vaddr_t *start, vaddr_t *end)
+{
+	uint32_t exceptions = thread_mask_exceptions(THREAD_EXCP_FOREIGN_INTR);
+	struct thread_core_local *l = thread_get_core_local();
```

Comment:
minor: prefer s/l/loc_thr/ or like, since `l` looks like `1` in many file editors.

Reply:
hmmm... `l` is used in other functions...
discard my comment.